packages/analytics-js/src/app/RudderAnalytics.ts

```diff
@@ -53,6 +54,8 @@ class RudderAnalytics implements IRudderAnalytics<IAnalytics> {
       return RudderAnalytics.globalSingleton;
       // END-NO-SONAR-SCAN
     }
+    // attach error listeners
```

Comment:
This comment doesn't add any value. The code is self-explanatory.
Did you mean to add anything else but forgot?

Reply:
Removed the comment